api/tests/MemberMatch/MemberMatchTests.cs

```diff
@@ -1,35 +1,25 @@
-using MemberMatch;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-
 namespace TestMemberMatch
 {
+    using System.Collections.Generic;
```

Comment:
these should be before the namespace (according to our stylecop I believe)

Reply:
StyleCop wanted the 'using' inside the namespace. https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1200.md

I don't know why it doesn't complain about the other files.

Reply:
Interesting... @luillo1  any idea why? You're the one that set stylecop up 😄